application/views/mobile/pages/sunglasses.php

```diff
@@ -82,6 +82,15 @@ class = "isInCartMark"
             </div>
         <?php endforeach; ?>
     </div>
+    <div id="mobile-shop-benefits-infographic">
+        <div class="benefits-container">
+            <img class="mobile-shop-benefits-infographic-icon" src="<?=IMG?>shop-benefits-infographic/5_years.svg" />
```

Comment:
would be great to use BEM instead: `mobile-shop-benefits-infographic__icon` (`__` before `icon` to indicate that it is a child of `mobile-shop-benefits-infographic`) (see http://getbem.com/naming/).